.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install libgl1-mesa-dev xorg-dev
```

Comment:
Are the first two whitespaces necessary? This should be equivalent this in yml:

```yaml
before_install:
- sudo apt-get -qq update
- sudo apt-get install libgl1-mesa-dev xorg-dev
```

Reply:
Read something about yml format.

Reply:
ok